test/plugin/base_test.rb

```diff
@@ -1025,17 +1025,26 @@ def test_http_request_partial_from_record
     end
   end
 
-  def test_http_request_without_referer_from_record
+  def test_http_request_from_record_with_referer_nil_or_absent
     setup_gce_metadata_stubs
-    setup_logging_stubs do
-      d = create_driver
-      d.emit('httpRequest' => http_request_message_without_referer)
-      d.run
-    end
-    verify_log_entries(1, COMPUTE_PARAMS, 'httpRequest') do |entry|
-      assert_equal http_request_message_without_referer, entry['httpRequest'],
-                   entry
-      assert_nil get_fields(entry['structPayload'])['httpRequest'], entry
+    {
+      http_request_message_with_referer_nil => \
+        http_request_message_expected_struct_payload,
+      http_request_message_with_referer_absent => 'null'
+    }.each do |message, expected|
+      @logs_sent = []
```

Comment:
My bad. I forgot to delete the one in the base test. It was definitely invalid. The test did not error out because both tests override it. I've just deleted it.